### ISSUE TITLE
Adding #UNK# to training set

### ISSUE THREAD

**Issue by woshibiantai** [jonbeibeibei/ML_Sentiment_Analysis#4]
One problem with estimating the emission parameters is that some words that appear in the test set do not appear in the training set. One simple idea to handle this issue is as follows. First, replace those words that appear less than k times in the training set with a special token #UNK# before training. This leads to a “modified training set”. We then use such a modified training set to train our model.

During the testing phase, if the word does not appear in the “modified training set”, we replace that word with #UNK# as well.

Set k to 3, implement this fix into your function for computing the emission parameters.
